src/WebJobs.Script/ScriptHostBuilderExtensions.cs

```diff
@@ -306,13 +313,7 @@ public static IHostBuilder AddScriptHostCore(this IHostBuilder builder, ScriptAp
                          .GetSection(ConfigurationSectionNames.Aggregator)
                          .Bind(o);
                     });
-                services.AddOptions<ScaleOptions>()
-                    .Configure<IConfiguration>((o, c) =>
-                    {
-                        c.GetSection(ConfigurationSectionNames.JobHost)
-                         .GetSection(ConfigurationSectionNames.Scale)
-                         .Bind(o);
-                    });
+                services.ConfigureOptions<ScaleOptionsSetup>();
```

Comment:
I think this looks right now. My main concern is/was whether we'd have any breaking changes with apps that might be using app settings to configure ScaleOptions now, if we're changing the config section names we're binding to. The SDK logic is [here](https://github.com/Azure/azure-webjobs-sdk/blob/474b704e2711977d5ea106f19bc060970b5c151e/src/Microsoft.Azure.WebJobs.Host/Hosting/WebJobsServiceCollectionExtensions.cs#L195) and it uses a helper that allows the Functions Host to override the section name which it does [here](https://github.com/Azure/azure-functions-host/blob/d207a4380e18aa79a7545762bd2f5107f8338893/src/WebJobs.Script/ScriptHostBuilderExtensions.cs#L396). So I think we're good, I just need you to be sure/confirm :)

Reply:
yes, there is no breaking change - the section names remain the same.